app/js/index.js

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
 import Router from 'react-router';
 
+import Navbar from './components/Navbar';
+import Event from './components/Event';
+
 const {Route, RouteHandler} = Router;
 
 class App extends React.Component {
   render() {
     return (
       <div>
-        <h1>asdf</h1>
+        <Navbar page="Home"/>
+        <div className="container">
+          <div className="row">
+            <div className="col-xs-4">
+              <h1>Stocks</h1>
+            </div>
+            <div className="col-xs-4">
+              <h1>Feed</h1>
 
+            </div>
+            <div className="col-xs-4">
+              <h1>Calendar</h1>
+              <Event title="R1 Release" day="Monday" startTime="12:00pm">
```

Comment:
Should we consider moving these to a json file so we can `GET` them (and pretend we are using a server?